test/framework/yeb.py

```diff
@@ -61,6 +61,8 @@ def test_parse_yeb(self):
         test_files = {
             'bzip2.yeb': 'bzip2-1.0.6-GCC-4.9.2.eb',
             'gzip.yeb': 'gzip-1.6-gcc-4.9.2.eb',
+            'intel-2015b.yeb': 'intel-2015b.eb',
```

Comment:
maybe it's better to create the `.yeb` equivalent of `ictce-4.1.13.eb`, which is already in the test easyconfigs

add `intel-2015b.eb` requires also adding easyconfigs for each of the components (`icc`, `ifort`, etc.) to make the tests happy